docs/sources/operations/autoscaling_queriers.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Autoscaling Loki Queriers
+weight: 30
+---
+
+# Autoscaling Loki Queriers
+
+A microservices deployment of a Loki cluster that runs on Kubernetes will typically handle a
+workload that varies throughout the day.
+To make Loki easier to operate and optimize the cost of running Loki at scale,
+we have designed a set of resources to help you autoscale your Loki queriers.
+
+## Prerequisites
+
+You need to run Loki in Kubernetes as a set of microservices.
+
+We recommend using [KEDA (*Kubernetes-based Event Driven Autoscaler*)](https://keda.sh/) to configure autoscaling
+based on Prometheus metrics. Refer to the [KEDA documentation](https://keda.sh/docs/latest/deploy) to learn more
+about setting up KEDA in your Kubernetes cluster.
+
```

Comment:
I think you forgot to mention in the prerequisites that users need to run the v2 of query frontend (as the metrics used for the scaling are only exposed from the query-scheduler)

Reply:
> the v2 of query frontend

@Raskyld Can you elaborate on what you mean by "v2" here?

@KMiller-Grafana should we add versioning requisites in these docs? IIUC, docs are coupled with the version (e.g. [docs for 2.4.x][1] and [docs for latest][2]).

[1]: https://grafana.com/docs/loki/v2.4.x/
[2]: https://grafana.com/docs/loki/latest/

Reply:
Of course : there is two implementations of the `query-frontend` ([namely v1, v2](https://github.com/grafana/loki/tree/main/pkg/lokifrontend/frontend)) the first one has a builtin query queue and the second one use `scheduler_worker` (see [here](https://github.com/grafana/loki/blob/main/pkg/lokifrontend/frontend/v2/frontend_scheduler_worker.go)) to forward queries to the schedulers. 

*tl; dr* If you use the query schedulers you're using query-frontend v2 otherwise you fallback to the v1.

The v1 typically doesn't have the metric you're using for the auto-scaling, hence, it would be wise to add the query scheduler setup in the requirements.

If you need more information do not hesitate !

Reply:
> there is two implementations of the query-frontend ([namely v1, v2](https://github.com/grafana/loki/tree/main/pkg/lokifrontend/frontend))

TIL. I'll add a note to the prerequisites. Thank you! 🚀

Reply:
You're very welcome! Thanks you :D